.github/workflows/pull-request.yml

```
@@ -251,21 +251,28 @@
         run: |
           npm run test -- --watchAll=false --coverage
 
+      - name: Upload Jest Coverage to Codecov
+        if: steps.changed-files.outputs.only_changed != 'true'
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.4)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)